cosmwasm/test/injective.ts

```diff
@@ -0,0 +1,717 @@
+import { execSync } from "child_process";
+import { Bech32, toHex } from "@cosmjs/encoding";
+import {
+  makeTransferVaaPayload,
+  signAndEncodeVaa,
+  TEST_SIGNER_PKS,
+} from "./src/helpers/vaa";
+import { keccak256 } from "ethers/lib/utils";
+
+function sleep(ms: number) {
+  return new Promise((resolve) => {
+    setTimeout(resolve, ms);
+  });
+}
+
+function createHashQueryString(hash: string) {
+  const txhashQuery = "injectived query tx ";
+  const jsonFlag = " -o json";
+  return txhashQuery + hash + jsonFlag;
+}
+
+function nativeToHex(address: string) {
+  return toHex(Bech32.decode(address).data).padStart(64, "0");
+}
+
+// wasm contracts
+const wormholeWasm: string = "wormhole.wasm";
+const tokenBridgeWasm: string = "token_bridge_terra_2.wasm";
+const cw20WrappedWasm: string = "cw20_wrapped_2.wasm";
+const mockBridgeWasm: string = "mock_bridge_integration_2.wasm";
+const cw20BaseWasm: string = "cw20_base.wasm";
+
+function createWasmFullPath(w: string) {
+  const pathToWasms: string =
+    "/home/pnoel/git/injective/wormhole/cosmwasm/artifacts/";
```

Comment:
can we avoid the full path here. not everyone will be pnoel 🙃

Reply:
Done